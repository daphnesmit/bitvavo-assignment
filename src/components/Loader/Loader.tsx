export const Loader = () => (
  <div className="mx-auto">
    <div
      className="inline-block h-6 w-6 animate-spin rounded-full border-[3px] border-current border-t-transparent text-blue-600 dark:text-blue-500"
      role="status"
      aria-label="loading"
    >
      <span className="sr-only">Loading...</span>
    </div>
  </div>
);
