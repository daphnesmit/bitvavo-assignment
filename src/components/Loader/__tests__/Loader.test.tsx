import { render } from '../../../tests/testUtils';
import { Loader } from '../Loader';

describe('Loader', () => {
  it('renders correctly', () => {
    const { container } = render(<Loader />);
    expect(container).toMatchSnapshot();
  });
});
