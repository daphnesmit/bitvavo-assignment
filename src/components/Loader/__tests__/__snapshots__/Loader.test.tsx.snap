// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<div>
  <div
    class="mx-auto"
  >
    <div
      aria-label="loading"
      class="inline-block h-6 w-6 animate-spin rounded-full border-[3px] border-current border-t-transparent text-blue-600 dark:text-blue-500"
      role="status"
    >
      <span
        class="sr-only"
      >
        Loading...
      </span>
    </div>
  </div>
</div>
`;
